anonymizer/OpenDiffix.Service/Program.fs

```diff
@@ -108,7 +108,7 @@ let handlePreview
   let mutable starBucket: Bucket option = None
   let starBucketHook = AggregationHooks.StarBucket.hook (fun bucket -> starBucket <- Some bucket)
 
-  let result = runQuery [ ledHook; starBucketHook ] query inputPath anonParams
+  let result = runQuery [ (* ledHook; *) starBucketHook ] query inputPath anonParams
```

Comment:
Can you also disable the `starBucketHook`? It doesn't seem stable enough to include it in a release.

Reply:
Yeah, good idea since we plan an immediate release.